benchmarks/benchmark-ab.py

```diff
@@ -133,6 +134,9 @@ def warm_up():
     
     execute(ab_cmd, wait=True)
 
+    global warm_up_lines
```

Comment:
Thanks, this is an interesting catch - could you perhaps refactor a bit more to remove the use of the global variable, to avoid confusion in the future? e.g., have the warm_up() function return the number of lines to be removed, and have the run_benchmark() command consume it.

Reply:
Hi @nskool, thanks for the hint, a1f2359 solves this